open-metadata-implementation/common-services/ocf-metadata-management/ocf-metadata-handlers/src/main/java/org/odpi/openmetadata/commonservices/ocf/metadatamanagement/handlers/SchemaTypeHandler.java

```diff
@@ -1059,7 +1059,7 @@ else if (instanceElementType.getElementOrigin() == ElementOrigin.LOCAL_COHORT)
             return true;
         }
 
-        return false;
+        return instanceElementType.getElementOrigin() == ElementOrigin.CONFIGURATION;
```

Comment:
Configuration is not local cohort

Reply:
Configuration is the default value when no other value is provided -- and unfortunately without this an external sourced entity creation is attempted (without any external metadata collection ID)...  For some reason this logic differentiating between external and local cohort is only present when trying to create schema attributes, and not schema types (not sure why they're different in this regard)?